project/Paradox.scala

```diff
@@ -23,26 +23,27 @@ import sbt._
 import scala.concurrent.duration._
 
 object Paradox {
-
+  val pekkoBaseURL = "https://pekko.apache.org"
   val propertiesSettings = Seq(
     Compile / paradoxProperties ++= Map(
-      "canonical.base_url" -> "https://doc.akka.io/docs/akka/current",
+      "canonical.base_url" -> s"$pekkoBaseURL/docs/akka/current",
       "github.base_url" -> GitHub
         .url(version.value), // for links like this: @github[#1](#1) or @github[83986f9](83986f9)
-      "extref.pekko.http.base_url" -> "https://doc.akka.io/docs/akka-http/current/%s",
-      "extref.pekko-management.base_url" -> "https://doc.akka.io/docs/akka-management/current/%s",
+      "extref.pekko.http.base_url" -> s"$pekkoBaseURL/docs/akka-http/current/%s",
+      "extref.pekko-management.base_url" -> s"$pekkoBaseURL/docs/akka-management/current/%s",
       "extref.platform-guide.base_url" -> "https://developer.lightbend.com/docs/akka-platform-guide/%s",
       "extref.wikipedia.base_url" -> "https://en.wikipedia.org/wiki/%s",
       "extref.github.base_url" -> (GitHub.url(version.value) + "/%s"), // for links to our sources
       "extref.samples.base_url" -> "https://developer.lightbend.com/start/?group=akka&amp;project=%s",
       "extref.ecs.base_url" -> "https://example.lightbend.com/v1/download/%s",
-      "scaladoc.pekko.base_url" -> "https://doc.akka.io/api/akka/2.6/org/apache",
-      "scaladoc.pekko.http.base_url" -> "https://doc.akka.io/api/akka-http/current/org/apache",
+      "pekko.doc.dns" -> s"$pekkoBaseURL",
+      "scaladoc.pekko.base_url" -> s"$pekkoBaseURL/api/akka/2.6/org/apache",
```

Comment:
any path with `/akka` in will need to change to `/pekko`

paths like `akka-http` will need to change to `pekko-http` -- likewise for `akka-management`, etc.